src/test-utils/dom/content-layout/index.ts

```diff
@@ -2,15 +2,28 @@
 // SPDX-License-Identifier: Apache-2.0
 import { ComponentWrapper, ElementWrapper } from '@cloudscape-design/test-utils-core/dom';
 import styles from '../../../content-layout/styles.selectors.js';
+import testutilStyles from '../../../content-layout/test-classes/styles.selectors.js';
 
 export default class ContentLayoutWrapper extends ComponentWrapper<HTMLDivElement> {
   static rootSelector: string = styles.layout;
 
   findHeader(): ElementWrapper | null {
-    return this.findByClassName(styles.header);
+    return this.findByClassName(testutilStyles.header);
   }
 
   findContent(): ElementWrapper {
     return this.findByClassName(styles.content)!;
   }
+
+  findNotifications(): ElementWrapper {
+    return this.findByClassName(styles.notifications)!;
```

Comment:
Why do you use test-utils styles for other methods but not here and not in breadcrumbs?

Reply:
Yes, because we want test-util selectors to be more stable regardless style changes

Reply:
See the changes `src/__tests__/__snapshots__/test-utils-selectors.test.tsx.snap`. There will be incompatibility in `findHeader` test util